src/locales/cy/pages.yml

```diff
@@ -99,7 +143,14 @@ rti-p60-statutory-maternity-pay:
       <p>Dim ond y swm o un P60 sydd angen i chi ei rhoi.</p>
 rti-p60-student-loan-deductions:
   title: Rhowch swm y ‘didyniadau benthyciad myfyriwr’ o'ch P60
-  inset: Defnyddiwch eich P60 o'r flwyddyn dreth {{ yearRangeStart }} i {{ yearRangeEnd }} ateb y cwestiwn hwn.
+  h1: Rhowch swm y ‘didyniadau benthyciad myfyriwr’ o'ch P60
+  inset:
+    html: >-
+      <p> Gallwch ddefnyddio P60 ar gyfer y naill un o'r blynyddoedd treth hyn: <p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>{{ currentYearRangeStart }} i {{ currentYearRangeEnd }}</li>
+        <li>{{ previousYearRangeStart }} i {{ previousYearRangeEnd }}</li>
+      </ul>
```

Comment:
To avoid having this HTML across both content files where it's harder to read/maintain - is it possible to still define the structure of this in the template and just the separate lines of content here?

Reply:
We can but then we would have to cater for every different inset block, some are one liners, some are bullet points such as this. At least here its a single place to change whereas the template would also have to be updated every time we add a new inset and we would have different if statements for the different variations in the template. I think any alternative way would convolute this

Reply:
Also its not uncommon to add html to the translations